PortalSwift/Sources/PortalSwift/Utils/Blockchains/PortalBlockchain.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  PortalBlockchain.swift
+//
+//
+//  Created by Blake Williams on 3/30/24.
+//
+
+import Foundation
+
+public class PortalBlockchain {
+  static let mainnetReferences: [PortalNamespace: String] = [
+    .eip155: "1",
```

Comment:
Do we need to add other mainnet chains here? (Not sure where this gets used yet)

Reply:
This is more for if we want to have some checks using the `isMainnet` value at some point in the future. For now, we're still only supporting Ethereum, but we should add Solana mainnet here when we add Solana support.